Dockerfile

```diff
@@ -29,10 +23,30 @@ RUN apt-get install -y libcouchbase3 \
     libcouchbase-dbg libcouchbase3-libev \
     libcouchbase3-libevent
 
+# Install kore.io
+RUN wget https://kore.io/releases/kore-4.1.0.tar.gz
+RUN wget https://kore.io/releases/kore-4.1.0.tar.gz.minisig
+RUN add-apt-repository ppa:dysfunctionalprogramming/minisign \
+    && apt-get install -y minisign
+RUN minisign -Vm kore-4.1.0.tar.gz -P RWSxkEDc2y+whfKTmvhqs/YaFmEwblmvar7l6RXMjnu6o9tZW3LC0Hc9 \
+    && tar -xvzf kore-4.1.0.tar.gz \
+    && cd kore-4.1.0 \
+    && make \
+    && make install \
+    && cp kore /usr/local/bin/kore \
+    && cp kodev/kodev /usr/local/bin/kodev
+
+RUN mkdir -p /try-cb-lcb
+
+ADD . /try-cb-lcb
+
+WORKDIR /try-cb-lcb
+
 RUN kodev clean && kodev build
 
 # Expose ports
 EXPOSE 8080
 
 # Set the entrypoint
-ENTRYPOINT ["./wait-for-couchbase.sh", "kore", "-n", "-r", "-c", "/try-cb-lcb/conf/try-cb-lcb.conf"]
+# ENTRYPOINT ["./wait-for-couchbase.sh", "kodev", "run"]
```

Comment:
Probably can get rid of the commented out line now?

Reply:
@osfameron @maria-robobug  More background... If using the `kore` command, I added the `-f` to the other invocation example (run in foreground) for easier debugging since a syslogger is not installed in the instance. Also need the `-r` and `-c` unless you plan to get all that special setup going as well (but that's overkill for this purpose). However, given the intent of this, perhaps it is easier to just use the `kodev run` command instead. I have not heartburn either way ultimately. ;)